modules/security-context-constraints-psa-opting.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-psa-opting_{context}"]
+= Controlling pod security admission synchronization
+
+You can enable or disable automatic pod security admission synchronization for most namespaces.
+
+[NOTE]
+====
+By default, namespaces that have the prefix 'openshift-' have pod security admission synchronization disabled. 
```

Comment:
@maxwelldb any namespace prefixed with "openshift" has the sync disabled by default
Therefore, for that namespace which are part of the OCP payload has no way of we make the label syncer work at all. These ones are mainly prefixed with "openshift" but we cannot say that is created by the installer because we created "openshift-operators" for example and this one is NOT part of the OCP payload then we need to clarify in IMO the best way is to link the list of those namespaces which are part of the OCP payload ( from the code ) and will never have the sync working at all.

Reply:
> So is it the case that, as a user, if I create a namespace called openshift-user-cool-namespace, sync is disabled by default?

yes

> These ones are mainly prefixed with "openshift" but we cannot say that is created by the installer because we created "openshift-operators"

"openshift-operators" is the only exception, you can therefore call it that but this one exception should be specifically called out.

I don't want a specific list of namespaces because I am not convinced about its stabiity.